pkg/config/config.py

```diff
@@ -1,4 +1,4 @@
-"""Config module for SAM (Serverless Application Model)"""
+"""Config module for SAM (Serverless Application Model)."""
```

Comment:
Nitpick I'd probably remove the reference to SAM since this holds stuff now like the `TIMEZONE` global.

Reply:
fixed.